rs/nns/governance/tests/governance.rs

```diff
@@ -9193,13 +9192,18 @@ fn test_update_node_provider() {
 
     let np = NodeProvider {
         id: Some(controller),
-        reward_account: Some(account.into_proto()),
```

Comment:
This seems tangential?

Reply:
Is part of removing dfn_protobuf dependency.  The PR chain can show all the little steps along the way